### PR TITLE
Irminger 10 recovered ingestion csvs

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: GI01SUMO/GI01SUMO_R00010_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00010/cg_data/dcl12/wavss/*.wavss.log,GI01SUMO-SBD12-05-WAVSSA000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00010/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GI01SUMO/R00010/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA001,recovered_host,Available,
+mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00010/instruments/dcl12/METBK-CT*/*.hex,GI01SUMO-SBD12-06-METBKA001,recovered_inst,Available,
+mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00010/cg_data/dcl12/fdchp/*.fdchp.log,GI01SUMO-SBD12-08-FDCHPA000,recovered_host,Available,
+mi.dataset.driver.fdchp_a.fdchp_a_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00010/instruments/dcl12/FDCHP_sn*/D*/fdchp_*.dat,GI01SUMO-SBD12-08-FDCHPA000,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GI01SUMO/GI01SUMO_R00010_ingest.csv:86]: looks good

File: GI02HYPM/GI02HYPM_R00010_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00010/instruments/CTDMO_sn*/*.hex,GI02HYPM-RIM01-02-CTDMOG039,recovered_inst,Available,
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00010/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-00-WFPENG000,recovered_wfp,Available,
+mi.dataset.driver.flord_l_wfp.flord_l_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00010/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-01-FLORDL000,recovered_wfp,Available,
+mi.dataset.driver.dosta_ln.wfp.dosta_ln_wfp_driver,/omc_data/whoi/OMC/GI02HYPM/R00010/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-03-DOSTAL000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00010/instruments/GWFP_sn*/0*/C*.DA?,GI02HYPM-WFP02-04-CTDPFL000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_l.wfp.vel3d_l_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00010/instruments/GWFP_sn*/0*/A*.DA?,GI02HYPM-WFP02-05-VEL3DL000,recovered_wfp,Available,
```
Comment (rtravis-whoi) [inline GI02HYPM/GI02HYPM_R00010_ingest.csv:11]: looks good

File: GI03FLMA/GI03FLMA_R00010_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00010/instruments/CTDMO_sn37-13447/*.hex,GI03FLMA-RIM01-02-CTDMOH051,recovered_inst,Available,
+mi.dataset.driver.sio_eng.sio.sio_eng_sio_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00010/cg_data/SSIOC_sn*/STATUS/STA*.DAT,GI03FLMA-RIS01-00-SIOENG000,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.sio.dosta_abcdjm_sio_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00010/cg_data/SSIOC_sn*/DOSTA/DOS*.DAT,GI03FLMA-RIS01-03-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00010/instruments/PHSEN_sn*/SAMI*.txt,GI03FLMA-RIS01-04-PHSENF000,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00010/cg_data/SSIOC_sn*/PHSEN/PHS*.DAT,GI03FLMA-RIS01-04-PHSENF000,recovered_host,Available,
+mi.dataset.driver.flort_dj.sio.flort_dj_sio_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00010/cg_data/SSIOC_sn*/FLORT/FLO*.DAT,GI03FLMA-RIS01-05-FLORTD000,recovered_host,Available,
```
Comment (rtravis-whoi) [inline GI03FLMA/GI03FLMA_R00010_ingest.csv:42]: looks good

File: GI03FLMB/GI03FLMB_R00010_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+#,/omc_data/whoi/OMC/GI03FLMB/R00010/cg_data/MSIOC_sn*/CTD/CTD*.DAT,GI03FLMB-RIM01-02-CTDMOG066,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes.
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/instruments/CTDMO_sn37-13428/*.hex,GI03FLMB-RIM01-02-CTDMOG066,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GI03FLMB/R00010/cg_data/MSIOC_sn*/CTD/CTD*.DAT,GI03FLMB-RIM01-02-CTDMOG067,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes.
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/instruments/CTDMO_sn37-13421/*.hex,GI03FLMB-RIM01-02-CTDMOG067,recovered_inst,Available,
+#,/omc_data/whoi/OMC/GI03FLMB/R00010/cg_data/MSIOC_sn*/CTD/CTD*.DAT,GI03FLMB-RIM01-02-CTDMOG068,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes.
+#mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/instruments/CTDMO_sn37-13623/*.hex,GI03FLMB-RIM01-02-CTDMOG068,recovered_inst,Not Available,No recovered data
```
Comment (rtravis-whoi) [inline GI03FLMB/GI03FLMB_R00010_ingest.csv:31]: checklist & raw data repo show recovered instrument data - I think this being commented out w 'not available/no recovered data' is a copy/paste error from R00009.

**Review by rtravis-whoi — CHANGES_REQUESTED** [review]
on FLMB CTDMOG068 - checklist & raw data repo show recovered instrument data - I think this being commented out w 'not available/no recovered data' is a copy/paste error from R00009.

File: GI03FLMB/GI03FLMB_R00010_ingest.csv [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/instruments/CTDMO_sn37-13457/*.hex,GI03FLMB-RIM01-02-CTDMOH071,recovered_inst,Available,
+mi.dataset.driver.sio_eng.sio.sio_eng_sio_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/cg_data/SSIOC_sn*/STATUS/STA*.DAT,GI03FLMB-RIS01-00-SIOENG000,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.sio.dosta_abcdjm_sio_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/cg_data/SSIOC_sn*/DOSTA/DOS*.DAT,GI03FLMB-RIS01-03-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/instruments/PHSEN_sn*/SAMI*.txt,GI03FLMB-RIS01-04-PHSENF000,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/cg_data/SSIOC_sn*/PHSEN/PHS*.DAT,GI03FLMB-RIS01-04-PHSENF000,recovered_host,Available,
+mi.dataset.driver.flort_dj.sio.flort_dj_sio_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00010/cg_data/SSIOC_sn*/FLORT/FLO*.DAT,GI03FLMB-RIS01-05-FLORTD000,recovered_host,Available,
```
Comment (rtravis-whoi) [inline GI03FLMB/GI03FLMB_R00010_ingest.csv:42]: correction to CTDMOG068 recovered_instr looks good

**Review by rtravis-whoi — APPROVED** [review]

File: GI01SUMO/GI01SUMO_R00010_ingest.csv (hunk 1 above)
Comment (cooleyky) [inline GI01SUMO/GI01SUMO_R00010_ingest.csv:86]: looks good

**Pull request merged**